serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -226,7 +226,7 @@ export class _GLTFMaterialExporter {
             return _cubicBezierCurve(t, P0.y, P1.y, P2.y, P3.y);
         }
 
-        let diffuse = babylonStandardMaterial.diffuseColor.toLinearSpace().scale(0.5);

```

Comment:
What's the reason for this change?

Reply:
Currently we're halving the base color of the material, when our specular->roughness conversion does not use it at all. 

It feels like an "eyeball" factor, and in our current case  leads to poor conversion between simple materials. We removed this from the Autodesk exporters a while back, and I think we should do the same here.

Reply:
I agree it's an eyeball factor, but there is no correct conversion. The fact that Autodesk exporters do it differently is not necessarily a good argument. I don't think their shader model is the same as ours. If I recall correctly, there were a number of models that we converted that looked much too bright without this factor.

Reply:
Currently, while being halfed, most results i've got are too dark. (to the extend where it's highly visible) , i think i tested around  .66 - .75 with reasonably results

Reply:
Sounds like this problem isn't as trivial as I was expecting. Lets keep the focus of this PR to just the vertex color changes, and we can revisit the material base color conversion in another PR after I do some more investigation. Ideally something similar to https://kcoley.github.io/glTF/extensions/2.0/Khronos/KHR_materials_pbrSpecularGlossiness/examples/convert-between-workflows-bjs/ using our current conversion math should allow us to get a better idea of what we're seeing.

Reply:
Is that link for conversion between standard and PBR or the original spec/gloss vs metal/rough? I thought we had already created a site that did standard to PBR somewhere.

Reply:
That one was the old spec/gloss to metal rough, I'm not sure where we hosted a standard to PBR conversion sample...

Reply:
Hmm, I'm pretty sure we added one or there would be no reason to do so since I'm already hosting it here: https://bghgary.github.io/glTF/convert-between-workflows-bjs/ which is the original one. I think it's this one: https://github.com/kcoley/glTF/tree/kcoley/standardMatToSpecGloss